### ISSUE TITLE
Y24-010 Options to implement the ANOSPP non-destructive extraction pipeline.

### ISSUE THREAD

**Issue by andrewsparkes** [sanger/General-Backlog-Items#373]
**Description**
We need to consider how the new ANOSPP non-destructive extraction (NDE) process will be implemented and create a backlog to implement the solution.

**Output**
A backlog of user stories to implement the NDE two step process for ANOSPP.

**Who the primary contacts are for this work**
Scott T
Emma D
PSD developers familiar with Bioscan

**Knowledge or Stake holders**
Scott T has a diagram for the ANOSPP options, here:
(updated to v3 on 18th March 2024)
[Non-Destructive Extractions BIOSCAN ANOSPP v3.pdf](https://github.com/sanger/General-Backlog-Items/files/14634989/Non-Destructive.Extractions.BIOSCAN.ANOSPP.v3.pdf)
See ANOSSP Confluence page for up to date links

**Additional context or information**
Best to meet with Scott T and Emma D to go through the details for this.

**Comment by SujitDey2022** [comment]
Team Discussion 05 March 2024:

1.  Not blocked by https://github.com/sanger/General-Backlog-Items/issues/372
2. The output of this research story will be estimates of implementing either of the 2 options proposed by Scott T.
3. Open ended questions related to technical implementation in Lab - Plate warping, Addition of controls,
4. Can the pipeline be made more flexible allowing the option to choose the steps involving transfer of plate?
5. This should be treated as a separate pipeline as it is not compatible with the current bioscan design.
6. Does the new Anospp pipeline design help simplify the Bioscan pipeline design with the Anospp not coming into the picture at Lysate plate prep stage? If yes, what could be changes required in Bioscan?

**Comment by andrewsparkes** [comment]
Updated process map to v3 on 18th March 2024.
From Scott:
After having observed the process in GSLP, I have revised the process map to better reflect the LIMS process.

PCR1 creation has a single step bed verification rather than 2-step.

**Comment by TWJW-SANGER** [comment]
Updating story to reduce the scope following discussions on 28/5/2024.

**Comment by andrewsparkes** [comment]
Stories created:
[Y24-135 - ANOSPP - Sample Manifest for creation of Stock plate](https://github.com/sanger/limber/issues/1737)
[Y24-136 - ANOSPP - Creation of configuration for submission and pipeline](https://github.com/sanger/limber/issues/1738)

**Comment by andrewsparkes** [comment]
@SujitDey2022 Would you like to make an **ANOSPP** issue label for these 2 stories above in SS and Limber?
And in terms of sizes I think safest to say the first is a medium and second a large, but I would hope it's less in reality.

**Comment by KatyTaylor** [comment]
@andrewsparkes stories look good. I left a comment on the manifest one.

I've just been reading through Scott's doc again and found a few things I thought worth mentioning:

1. "New LBSN-equivalent plates will require a 2x label barcode format similar to BIOSCAN LBSN-96 Lysate plates (which includes the Partner ID + -SDC suffix on the second label)." - is this part of Y24-136?
2. Fake cherrypicking before WGS submission - this was something that they wanted to improve upon - I think this is covered already as https://github.com/sanger/sequencescape/issues/4127 is in the backlog and labelled as 'ANOSPP', so can be done as part of this group of stories.
3. WGS manifest - SSR/R&D would prefer that the new manifest for registering samples that have previously undergone ANOSPP GBS validates that H12 is empty - is this in scope? If so, would need another story.

**Comment by andrewsparkes** [comment]
Have updated Y24-136 to include requirement for double label.

And asked Scott to clarify whether we need the empty H12 validation on the manifest upload.

**Comment by andrewsparkes** [comment]
Added a new story for the H12 empty manifest type requirement:

[Y24-139 - ANOSPP - Add a manifest template for WGS with a restriction on use of H12](https://github.com/sanger/sequencescape/issues/4146)

So, including the 2 above plus the one in the backlog about fake cherrypicking and this one, that is 4 stories in total.
